Migration.md

```diff
@@ -7,6 +7,8 @@ release will remove the deprecated code.
 
 ## Ignition Gazebo 5.x to 6.x
 
+* Marker example has been moved to Ignition GUI.
```

Comment:
It will eventually be more than just the example. We should deprecate the `MarkerManager` class in this repository too :sparkles: